configurations/operator/perf-regression-latency-and-throughput-multitenant-2-clients.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 540
+user_prefix: 'perf-regrn-lat-and-thpt-2-clients'
+
+# NOTE: deploy 8 K8S nodes of the 'loader' type and then create 4 loader pods per DB cluster
+n_loaders: 8
+k8s_n_loader_pods_per_cluster: 4
+
+prepare_write_cmd: [[
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..30000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=30000001..45000000",
+    "cassandra-stress write no-warmup cl=ALL n=15000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=20 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=45000001..60000000",
+], [
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=25 -pop seq=1..7500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=25 -pop seq=7500001..15000000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=25 -pop seq=15000001..22500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=25 -pop seq=22500001..30000000",
```

Comment:
Please change the Cl=QUIRUM to cl=ALL here.

Reply:
Done